auth/basic/src/main/java/org/trellisldp/auth/basic/BasicAuthFilter.java

```diff
@@ -80,6 +87,7 @@ public BasicAuthFilter(final File file) {
     public BasicAuthFilter(final File file, final String realm) {
         this.file = file;
         this.challenge = "Basic realm=\"" + realm + "\"";
+        LOGGER.info("Configured BASIC Authentication Filter.");
```

Comment:
Adding logging here is fine, though I'd recommend setting it at `DEBUG` level. Ditto for the other logging methods.

Reply:
I disagree-- this is initialization code, performed only once. I think it would nice to see a single positive message for each component that successfully initializes.

Reply:
Yes, @ajs6f makes a good point here. Let's leave it this at `INFO` level.